.github/workflows/build.yml

```diff
@@ -35,5 +35,5 @@ jobs:
     - name: 'Upload Artifacts'
       uses: actions/upload-artifact@v4
       with:
-        name: build_${{matrix.build_type}}
+        name: build_${{matrix.platform}}_${{matrix.build_type}}
```

Comment:
It looks like it didn’t work. [The CI logs say](https://github.com/kevinbentley/Descent3/actions/runs/8729169956/job/23950607961?pr=50#step:6:18):

```
Artifact build__Debug has been successfully uploaded! Final size is 5014487 bytes. Artifact ID is 1423817003
```

Shouldn’t it say “build_windows_Debug” or something like that?

Reply:
Yeah, let me try that again.